Sources/NIOHTTP1/HTTPTypes.swift

```diff
@@ -18,6 +18,8 @@ let crlf: StaticString = "\r\n"
 let headerSeparator: StaticString = ": "
```

Comment:
please change the allocation counter test limits and reduce according to the number of allocs saved

Reply:
@weissi let me see if it changes anything (as it will only if we actual have a connection header).

Reply:
@weissi @Lukasa ok no changes as we never have such a header.